dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -2202,6 +2205,12 @@
           "items": {
             "$ref": "#/definitions/SummaryResultItem"
           }
+        },
+        "chapters": {
```

Comment:
Is it possible to merge this into the existing `summaries` (e.g. modify `#/definitions/SummaryResultItem` instead) such that our output schema doesn't change between scenarios?

Reply:
SummaryResultItem has 1 aspect and 1 text. I cannot think of an easy way to merge chapter into it..

Reply:
Can we add an optional `subtopics: array[SummaryResultItem]` field and extend/reuse the `context` field to include chapter metadata?

Reply:
SummaryResultItem only has 1 aspect and 1 text. For the chapter we produce both narrative and chapterTitle, there is no place to hold 2 texts. Also, the context is only point to a range in one conversationItem. SummaryResultItem reference an array of context as contexts. The current design of summaries in summarization result is summary text oriented, rather than structure oriented, so hard to extend.